proto/cosmos/base/store/v1beta1/listening.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package cosmos.base.store.v1beta1;
+
+option go_package = "github.com/cosmos/cosmos-sdk/store/types";
+
+// StoreKVPair is a KVStore KVPair used for listening to state changes (Sets and Deletes)
+// It optionally includes the StoreKey for the originating KVStore and a Boolean flag to distinguish between Sets and Deletes
+message StoreKVPair {
+  string store_key = 1; // the store key for the KVStore this pair originates from
+  bool set = 2; // true indicates a set operation, false indicates a delete operation
```

Comment:
```suggestion
  bool delete = 2; // false indicates a set operation, true indicates a delete operation
```

I suggest we flip this so the default is set and you need to explicitly set to mark a delete.

Reply:
Yeah that makes more sense, adjusted this